pulsar-broker/src/main/java/com/yahoo/pulsar/broker/namespace/OwnershipCache.java

```diff
@@ -130,9 +127,16 @@
                             if (LOG.isDebugEnabled()) {
                                 LOG.debug("Successfully acquired zk lock on {}", namespaceBundleZNode);
                             }
+                            final OwnedBundle bundle = new OwnedBundle(
+                                    ServiceUnitZkUtils.suBundleFromPath(namespaceBundleZNode, bundleFactory));
+                            // update the version
+                            localZkCache.getZooKeeper().exists(path, false, (rc1, path1, ctx1, stat) -> {
```

Comment:
what if broker-1 which owns bundle and due to zk session is lost ephemeral node is deleted and new ephemeral node will be created by other broker-2. 
now pervious broker-1 restarts and it still has bundle in cache so, while unloading bundle it deletes the znode. So, next time bundle can be loaded by broker-3. So, now same bundle can be owned by broker-2(existing producer-consumers still serving) and broker-3.